.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,26 @@
+name: Differential ShellCheck
+on:
+  pull_request:
+    branches: [main]
```

Comment:
This would equally apply to stable RHEL branches, so this should be dropped.

Reply:
I have updated the branches array to: `branches: [main, rhel-*]`. It should also cover RHEL branches. I think it's better to try to specify branches rather than run a check on all branches and possibly consume sources.

Reply:
ack, thank you! That's a good compromise.